routing/index_router.cpp

```diff
@@ -360,7 +360,7 @@ IRouter::ResultCode IndexRouter::AdjustRoute(m2::PointD const & startPoint,
                                     false /* strictForward */),
       graph);
 
-  AStarProgress progress(0, 100);
+  AStarProgress progress(0, 95);
```

Comment:
Could you please explain why do we need this? I don't see explaining comment or Jira ticket.